support-workers/src/typescript/model/sendAcquisitionEventState.ts

```diff
@@ -11,96 +11,50 @@ import {
 	userSchema,
 } from './stateSchemas';
 
+const commonSchema = z.object({
+	user: userSchema,
+	product: productTypeSchema,
+	productInformation: productPurchaseSchema,
+	paymentMethod: paymentMethodSchema,
+	paymentSchedule: paymentScheduleSchema,
```

Comment:
Because this is a common field it will now be present for Contributions which it wasn't before.